config/base/tekton-config.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tekton-config
```

Comment:
lets us name this config map `config-defaults`.

Reply:
in the case of openshift, this would be created in the `openshift-operators` namespace then should we name it `tekton-config-defaults`? assuming there would be other operators in that ns and `config-defaults`  is generalised name? wdyt

Reply:
sure